recipes-ni/sysconfig-settings/sysconfig-settings_1.0.bb

```diff
@@ -59,5 +59,17 @@ do_install () {
 }
 
 pkg_postinst_ontarget_${PN} () {
+	# Add the memory reservation for Base. We assert that if the section exists,
+	# it's already been added.
+	if ! grep -q 'RtLinuxMemReserve' /etc/natinst/share/ni-rt.ini; then
+		cat >> /etc/natinst/share/ni-rt.ini <<EOF
+[RtLinuxMemReserve]
+Base=24
+EOF
+	fi
+
+	# Enable ssh by default
+	/usr/local/natinst/bin/nirtcfg -s section=systemsettings,token=sshd.enabled,value=True
+
```

Comment:
I assume this was unique to the SystemLink DKMS image, because SL requires SSH to be enabled for target configuration. But non-SL targets - I think - have operated on the assumption that SSH will not be enabled by default. So we should discuss as a group whether or not it is acceptable to change that assumption. I think the answer is yes; but we might want to organize some communication to that effect.

Reply:
Hasn't that assumption already been changed, given the SystemLink DKMS image is installed as the OneRT image on all OneRT-supported targets?

Reply:
Good point. I'm not sure if it'll be evident to clients/customers that ssh is enabled by default. Maybe something that needs to go in release notes for customers.. and a post on General channel for internal clients?

Reply:
>Hasn't that assumption already been changed, given the SystemLink DKMS image is installed as the OneRT image on all OneRT-supported targets?

I... don't know. And that's concerning.

@chaitu236 could you bring this up to James and Brian either sideband on Teams or otherwise once Austin has stabilized. We need to ensure that whatever changes we make here are intentional.

In the meantime, I think it's probably true that this is was the case for the x64 OneRT images, so this review shouldn't be blocked.